### ISSUE TITLE
Raw result

### ISSUE THREAD

**Issue by chenshihfang** [MIV-XJTU/ARTrack#11]
Hi, could you please provide the raw 
evaluation results for ViT-B and ViT-L across all datasets tested. In addition, do you intend to make the pre-trained weights for ViT-L available for use soon?

Additionally, could you specify the number of GPUs utilized for ViT-B and ViT-L computations?

**Comment by AlexDotHam** [comment]
Since I merged two codes into one, the checkpoints I previously trained may no longer apply (for example, the accuracy of my ViT-L was actually trained with a prompt length of 3. Due to the time pressure for the paper submission, the subsequent ablation studies showing that increasing prompt length can effectively improve accuracy were conducted on ViT-B 256), so I may not be able to provide checkpoints that match this code. If you need them, I can provide the raw results (you should be able to download the validation data for GOT-10k and TrackingNet directly from the webpage). Please feel free to contact me by email.

**Comment by AlexDotHam** [comment]
When I trained ViT-B 256 and ViT-B 384, I used 4 cards and 8 cards of 3090 for training respectively. For training ViT-L, I used 4 cards of A6000 for training. As for the pretraining of ViT-L, since I really don't have server resources for training now, you may have to do it yourself.
